src/lib/public/copy/copy.ts

```diff
@@ -0,0 +1,9 @@
+import { toast, ToastType } from '$lib/components/toaster/toasts';
+
+/** @see <https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText> */
```

Comment:
original documentation for this wasnt good, document the fact that this copies text to the clipboard and then displays a notification or error message

Reply:
just added some comment to explain what this function do